### ISSUE TITLE
[BUG] Multipage app dcc.Store is bugging page switching

### ISSUE THREAD

**Issue by infwisniewski** [plotly/dash#2107]
Envs:

```
dash                          2.4.1
dash-bootstrap-components     1.1.0
dash-core-components          2.0.0
dash-extensions               0.1.3
dash-html-components          2.0.0
dash-table                    5.0.0

    - Ubuntu 22.04 LTS
    -Mozilla Firefox Browser 101.0.1 (64-bit)

**Describe the bug**

When using dcc.Store as first item in layout in multi-paged apps with app.py as url dispatcher,
when I change href via dbc.NavLink and dcc.Location recieve an input in page.layout returning callback :
Page is being changed for a few miliseconds and then it's switched back to original point.

In short : dcc.Store as first item is breaking page changing via dcc.Location as input.

**Expected behavior**

Page should switch as it's adress changes.

**Comment by AnnMarieW** [comment]
Hi @infwisniewski Would you be able to include a minimal example that reproduces the error?

**Comment by gvwilson** [comment]
Hi - we are tidying up stale issues and PRs in Plotly's public repositories so that we can focus on things that are most important to our community. If this issue is still a concern, please add a comment letting us know what recent version of our software you've checked it with so that I can reopen it and add it to our backlog. (Please note that we will give priority to reports that include a short reproducible example.) If you'd like to submit a PR, we'd be happy to prioritize a review, and if it's a request for tech support, please post in our community forum. Thank you - @gvwilson
